integreat_cms/cms/models/users/organization.py

```diff
@@ -28,7 +28,6 @@ class Organization(AbstractBaseModel):
         verbose_name=_("logo"),
         on_delete=models.SET_NULL,
```

Comment:
```suggestion
        on_delete=models.PROTECT,
```
I guess if we want to enforce logos, we have to protect them from being deleted?